CONTRIBUTING.md

```diff
@@ -277,8 +294,11 @@ source env/bin/activate
 #  Linux: http://pillow.readthedocs.org/en/3.0.x/installation.html#linux-installation
 # install rest of dependencies
 pip install -r requirements.txt
+#In Ubuntu, if ImportError: No module named flask.ext.github occurs, it means the dependencies in requirement.txt is installed in /usr/lib instead of <project_repository>/env/lib/python<version>/site-packages.
```

Comment:
nit: can we get a space after `#` for this and other lines?
nit: please change to "are installed" (rather than "is installed")

s/requirement.txt/requirements.txt/